src/test/java/org/broadinstitute/consent/http/service/EmailServiceTest.java

```diff
@@ -106,10 +131,81 @@ public void testSendDataCustodianApprovalMessage() {
         }
     }
 
+    @Test
+    public void testSendNewResearcherEmail() throws MessagingException {
+        initFakeService();
+        User user = new User();
+        user.setUserId(1234);
+        user.setDisplayName("John Doe");
+
+        User so = new User();
+        user.setEmail("fake_email@asdf.com");
+        try {
+            service.sendNewResearcherMessage(user, so);
+        } catch (Exception e) {
+            fail("Should not fail sending message: " + e);
+        }
+
+        verify(sendGridAPI, times(1)).sendDatasetDeniedMessage(any(), any());
+
+        verify(emailDAO, times(1)).insert(
+                eq("1234"),
+                eq(null),
+                eq(1234),
+                eq(EmailType.NEW_RESEARCHER.getTypeInt()),
+                any(),
+                eq(
+                """
+                <!DOCTYPE html>
+                <html xmlns="http://www.w3.org/1999/xhtml" style="font-family: 'Roboto', sans-serif ;">
```

Comment:
I would lean towards not checking for the actual email content - this leads to brittle testing. The service call really only cares that it send **_some_** message to sendgrid. Looking at the contents of the message is going beyond the boundaries of the tested method call. If you want to test the actual content, we can do that in the test that creates the message itself, i.e. the template helper call.

Reply:
Hmm, sure. I can move this to a template helper call; I think here makes sense, too, but I don't feel strongly. I do, however, feel strongly that we do check it - before we never did check message content and I always felt weird about adding new emails / changing template content.

Reply:
> I do, however, feel strongly that we do check it

I'm not advocating for NOT checking it, I'm advocating for respecting the boundaries of the current method under test. `service.sendNewResearcherMessage` does not care what the content is, it just delegates the message to another class. The place for testing the html content is the method that generates that content which is `templateHelper.getNewResearcherTemplate`